__tests__/test.js

```diff
@@ -45,3 +105,79 @@ describe('getOrThrow', () => {
         expect(getOrThrow('TIMEZONE')).to.equal(process.env.TIMEZONE);
     });
 });
+
+describe('createIdObject', () => {
+    it('should create a object id', () => {
+        const returnObject = createIdObject(
+            ['mockedUser', 'mockedUser@user.com'],
+            mockedContributed
+        );
+
+        expect(returnObject.alternateId).to.equal('mockedUser@user.com');
+        expect(returnObject.github).to.equal('mockedUser');
+
+        const contributionsObject = [
+            { id: '1', type: 'IssueCommentEvent' },
+            { id: '2', type: 'PullRequestEvent' },
+        ];
+
+        contributionsObject.forEach((contribution, index) => {
+            expect(returnObject.contributions[index].id).to.equal(contribution.id);
+            expect(returnObject.contributions[index].type).to.equal(contribution.type);
+        });
+    });
+});
+
+describe('parseDatesFromArgv', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
+            CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID: '1',
+            GITHUB_IMPORTANT_EVENTS:
+                'CommitCommentEvent,IssueCommentEvent,IssuesEvent,PullRequestEvent,PullRequestReviewEvent,PullRequestReviewCommentEvent',
+        };
+
+        process.argv[2] = '2020-01-01';
+        process.argv[3] = '2020-12-01';
+    });
+
+    it('should generate 2 dates based on arguments', () => {
+        const moments = parseDatesFromArgv();
```

Comment:
Generally when I see `moment` as a variable, I assume it contains a moment object, or other non-string representation of an instance in time. Maybe `parsedDatesAsStrings` would be a clearer name?

Reply:
These are, in fact, moment objects so I'm going to leave this as-is